tests/fuzzers/runtime/runtime_fuzz.go

```diff
@@ -14,23 +14,23 @@
 // You should have received a copy of the GNU Lesser General Public License
 // along with the go-ethereum library. If not, see <http://www.gnu.org/licenses/>.
 
-// +build gofuzz
-
 package runtime
 
+import (
+	"github.com/XinFinOrg/XDPoSChain/core/vm/runtime"
+)
+
 // Fuzz is the basic entry point for the go-fuzz tool
 //
 // This returns 1 for valid parsable/runable code, 0
 // for invalid opcode.
 func Fuzz(input []byte) int {
-	_, _, err := Execute(input, input, &Config{
-		GasLimit: 3000000,
+	_, _, err := runtime.Execute(input, input, &runtime.Config{
+		GasLimit: 12000000,
 	})
-
 	// invalid opcode
 	if err != nil && len(err.Error()) > 6 && string(err.Error()[:7]) == "invalid" {
```

Comment:
can we remove `string(...)` as in https://github.com/ethereum/go-ethereum/pull/22029/files#diff-15232ffcfa7e7a617c8ab4e03e853454b9ed41fd854ef1d690f44bd2c8268f32R32

Reply:
yes